### PR TITLE
Sort Config Alphabetically for the Eyes 👀

### PR DESCRIPTION
**Pull request by maxime50**
This PR sort the configs on the following files:
- README.md
- lua/default-config.lua
- lua/settings.lua
- utils/installer/lv-config.example.lua

Also removed
``` lua 
O.plugin.hop.active = true
O.plugin.lazygit.active = true
O.plugin.markdown_preview.active = true
```
because these plugins are no longer built-in (https://github.com/ChristianChiarulli/LunarVim/pull/736#discussion_r664779248)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,32 +46,32 @@ Example:
...
-O.timeoutlen = 100
 O.leader_key = ' '
+O.relative_number = false
+O.timeoutlen = 100
 
 -- After changing plugin config it is recommended to run :PackerCompile
-O.plugin.hop.active = true
 O.plugin.colorizer.active = true
-O.plugin.trouble.active = true
+O.plugin.hop.active = true
```
Comment (abzcoding) [inline README.md:56]: while you're changing this I would appreciate it if you remove the following as they have been removed from built-in plugins:
```
O.plugin.hop.active = true
O.plugin.lazygit.active = true
O.plugin.markdown_preview.active = true
```

Comment (maxime50) [reply README.md]: fixed

**Comment by rebuilt** [comment]
Are you doing this manually or with the :sort command?

**Comment by maxime50** [comment]
> Are you doing this manually or with the :sort command?

I did it manually

**Comment by rebuilt** [comment]
Press 'Shift+v' to select a line.  Move the selection down to where you want it.  Press ":"  the colon to enter ex mode.  Type 'sort'.

**Comment by rebuilt** [comment]
You can also select a number of lines "Shift+v" + "2j" + ":sort"

**Comment by maxime50** [comment]
> You can also select a number of lines "Shift+v" + "2j" + ":sort"

Wow good to know thanks for this info. It would've been much faster 😅

**Comment by maxime50** [comment]
I double checked with the `:sort` method. Looks like everything's good now.

**Comment by maxime50** [comment]
Also sorted the config file because why not 🤷

**Review by Tastyep — APPROVED** [review]

**Pull request closed without merging**